internal/clientconn/conn.go

```diff
@@ -198,7 +198,43 @@ func (c *conn) run(ctx context.Context) (err error) {
 	for {
 		var reqHeader *wire.MsgHeader
 		var reqBody wire.MsgBody
+		var resHeader *wire.MsgHeader
+		var resBody wire.MsgBody
+		var validationErr *wire.ValidationError
+
 		reqHeader, reqBody, err = wire.ReadMessage(bufr)
+		if err != nil && errors.As(err, &validationErr) {
+			res := new(wire.OpMsg)
```

Comment:
I believe that we have a couple of lines in the code with this syntax:
```suggestion
			var res wire.OpMsg
```

Reply:
Fixed.